server/index.js

```diff
@@ -54,9 +51,7 @@ app.get('/products/:productItem', function (req, res) {
   });
 });
 //getting individual wishlist
-//why am i still alive...
 app.get('/wishlists/:username', function (req, res) {
-  //(productItem, callback)
   models.getWishlistByUsername(req.params.username, (err, data) => {
     if (err) {
       throw err;
```

Comment:
Maybe send a response back that an error has occurred, because I think your server will continue to handle the request until it ends/sends a response.